os_site/src/sass/style.scss

```diff
@@ -0,0 +1,202 @@
+@import "@texastribune/queso-ui/assets/scss/1-settings/all";
+@import "@texastribune/queso-ui/assets/scss/2-tools/all";
+@import "@texastribune/queso-ui/assets/scss/3-resets/all";
+@import "@texastribune/queso-ui/assets/scss/4-elements/all";
+@import "@texastribune/queso-ui/assets/scss/5-typography/all";
+@import "@texastribune/queso-ui/assets/scss/6-components/all";
+@import "@texastribune/queso-ui/assets/scss/utilities/all";
+
+.banner {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+
+  .banner-items {
+    max-width: 30%;
+  }
+}
+
+footer {
```

Comment:
Ashley (who is much better at CSS than I am!) might have a better, different opinion, but I typically hesitate to apply styles directly to HTML tags (like `footer` here or `main` below). You never when you'll need to style those tags differently on a particular page, and it can lead to some messy override CSS.

Reply:
Good tip! I also noticed that you can probably even get rid of all the footer styling [here](https://github.com/texastribune/texastribune.github.io/blob/7f1fb90986fc4a2ac58374ae62037b7dec36e1af/os_site/src/sass/style.scss#L75-L81) and [here](https://github.com/texastribune/texastribune.github.io/blob/7f1fb90986fc4a2ac58374ae62037b7dec36e1af/os_site/src/sass/style.scss#L20-L25) and maybe just use `has-xs-padding` [here](https://github.com/texastribune/texastribune.github.io/blob/7f1fb90986fc4a2ac58374ae62037b7dec36e1af/os_site/src/_includes/_footer.njk#L1). There's something (maybe the 70px height) causing a thin white line below the footer for me in FF and Chrome and simplifying that part seems to get rid of it.
![Screen Shot 2021-03-09 at 1 42 23 PM](https://user-images.githubusercontent.com/2974713/110528076-552be580-80dd-11eb-8f6f-042c7490243b.png)

Reply:
@AndrewGibson27 Thank you for this tip! Good to know to avoid that in the future. 
@ashley-hebler And great catch, thank you so much! This is a lot cleaner.